fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -2787,11 +2788,19 @@ void MultiVersionApi::runNetwork() {
 		});
 	}
 
-	localClient->api->runNetwork();
+	try {
+		localClient->api->runNetwork();
+	} catch (const Error& e) {
+		closeTraceFile();
+		throw e;
+	}
 
 	for (auto h : handles) {
 		waitThread(h);
 	}
+
+	TraceEvent("MVCRunNetworkTerminating");
```

Comment:
I think elsewhere we use "MultiVersion":

```suggestion
	TraceEvent("MultiVersionRunNetworkTerminating");
```